varya/style-rtl.css

```diff
@@ -2824,8 +2824,8 @@ nav a {
 }
 
 .site-logo .custom-logo {
-	border-radius: var(--branding--logo--border-radius);
 	max-width: var(--branding--logo--max-width);
+	max-height: var(--branding--logo--max-width);
```

Comment:
Is this left over from some other PR? (Also, I think that should be `var(--branding--logo--max-height)`)

Reply:
Ugh yes I think there was an instance where I failed to check in the compiled rtl styles, and this is leftover from that.

Reply:
Ah actually this is a bug in https://github.com/Automattic/themes-workspace/blob/2f0d972bca1914f38b12bc7dc3aec5883ef676f9/varya/assets/sass/components/header/_header-branding.scss#L65